App/src/components/Nabvar.js

```diff
@@ -36,9 +43,7 @@ function Nabvar() {
           </Link>
         </li>
         <li css={{ marginLeft: "auto" }}>
-          <Link css={linkStyle} to="/login">
-            Logout
-          </Link>
+          <button onClick={handleLogoutClick}>Logout</button>
```

Comment:
What happened with the `   <Link css={linkStyle} to="/login">`

Reply:
I deleted that by the validation of the login, it was that if there is no user redirects it to the login so the link was unnecessary.

Reply:
Thank you for your kind response, have a good day : ) !